src/Tribe/Views/V2/Hooks.php

```diff
@@ -254,6 +253,36 @@ function () {
 
 		// If there are any views checked, then run the filter.
 		add_filter( 'tribe_events_event_schedule_details_formatting', [ $this, 'handle_end_time_visibility' ] );
+
+		// Hook to add the flag for month view template.
+		add_action(
+			'tribe_template_pre_html:events/v2/month/calendar-body/day/calendar-events/calendar-event/date',
+			[ $this, 'handle_template_hide_end_time' ],
+			10,
+			4
+		);
+
+		// Once we are setup, broadcast ourself for further integrations.
+		do_action( 'tec_events_views_v2_hide_end_time_init', $this );
```

Comment:
Do we need a doc block for this?

Reply:
Yes please

Reply:
Passing the Hooks object feels _weird_ here?

Reply:
@Camwyn refactored a bit. This is so PRO can add it's own templates to the list that should be included.